docs/contributor-guide.md

```diff
@@ -27,6 +27,9 @@ Most contributions require you to agree to a
 Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
 the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
 
+For more information on our release channels and where your code should proabably start out,
```

Comment:
Could we have a header on this, to bring attention to it?  Just so if someone is looking for "release Channels" specifically they don't have to scan the page too hard?